CADETPythonSimulator/coupling_interface.py

```diff
@@ -8,20 +8,17 @@ class CouplingInterface(abc.ABC):
 
     @abc.abstractmethod
     def get_coupled_state(
-        self,
-        origin_list: list[(dict, float)],
-        state: str
-        ) -> np.ndarray:
+        self, origin_list: list[(dict, float)], state: str
+    ) -> np.ndarray:
```

Comment:
Add trailing comma to avoid line break. (See https://docs.astral.sh/ruff/settings/#format_skip-magic-trailing-comma).